docs/generators/javascript-flowtyped.md

```diff
@@ -7,6 +7,7 @@ sidebar_label: javascript-flowtyped
 | ------ | ----------- | ------ | ------- |
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
 |ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
+|enumPropertyNaming|Naming convention for enum properties: 'camelCase', 'PascalCase', 'snake_case', 'UPPERCASE', and 'original'| |PascalCase|
```

Comment:
why is the javascript-flowtyped affected?

Reply:
I was surprised also.
But apparently,
```
class JavascriptFlowtypedClientCodegen extends AbstractTypeScriptClientCodegen
```

Reply:
ah, ok alright